### DISCUSSION TITLE
[Project Beta] keyboard shortcuts not working

### DISCUSSION THREAD

**Discussion by Sammeeey** [community/community#12982] [Projects and Issues]
When I am in a project click on the small arrow icon on the left side of a task, it shows shortcuts:
- *Archive* [e]
- *Delete from project* [del]

Unfortunately these shortcuts don't work for me.

If I select a task (so that it has a blue border) and press the [del] key, I get the error message
*Title cannot be deleted*.
Using the [e] doesn't do anything.

**Comment by quicoto** [comment]
Same here

**Comment by krry** [comment]
Attempting this on Safari, latest iOS on iPad with no luck. Would love to quickly archive a list of draft issues.
